### PR TITLE
support storage analysis with statistics (#215)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #215

### CODE REVIEW

File: states/segment_anaylsis.go [hunk 1]
```diff
@@ -43,26 +44,46 @@ func (app *ApplicationState) StorageAnalysisCommand(ctx context.Context, p *Stor
...
+						binlog.LogPath, hrSize(info.Size)), p)
 				}
 				size += info.Size
 			}
-			fmt.Printf("Total binlog size: %s\n", hrSize(size))
+			MaybeAppendOutput(&output, fmt.Sprintf("Total binlog size: %s\n", hrSize(size)), p)
 		}
+		totalBinlogCount += len(segment.GetBinlogs())
+		totalDeltaLogCount += len(segment.GetDeltalogs())
+		totalStatsLogCount += len(segment.GetStatslogs())
```
Comment (congqixia) [inline states/segment_anaylsis.go:72]: this is the number of FieldBinlog not actually the Binlog number

File: states/segment_anaylsis.go
Comment (congqixia) [inline states/segment_anaylsis.go:1]: It's recommended to modify `show segment` command to list binlog/statslog/deltalog file number

File: states/segment_anaylsis.go (hunk 1 above)
Comment (MrPresent-Han) [reply states/segment_anaylsis.go]: changed

File: states/segment_anaylsis.go
Comment (MrPresent-Han) [reply states/segment_anaylsis.go:1]: changed

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**